src/main/java/org/jabref/gui/push/AbstractPushToApplication.java

```diff
@@ -34,6 +34,8 @@ public AbstractPushToApplication(DialogService dialogService) {
         this.dialogService = dialogService;
     }
 
```

Comment:
Please put the method body in
If you use eclipse or intellij you can import our code formatting styles. We have a config folder where they are

Reply:
done